explore/src/main/scala/explore/config/FiltersMultiSelect.scala

```diff
@@ -1,88 +1,88 @@
 // Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
-package explore.config
+// package explore.config
```

Comment:
I didn't want to spend time updating a control that we aren't currently using. If we decide to use it, we can come back to this commit in history to figure out how to revive it.